### ISSUE TITLE
Sequential playlist with repeatable parts dosn't work correctly

### ISSUE THREAD

**Issue by Ferz0id** [AzuraCast/AzuraCast#3842]
Can't clear one playlist without the others

**Using Docker installation method**
 Yes 

**AzuraCast version**
v0.12 Stable

**Describe the bug**

This problem came up when I put in my playlist all repeated items, jingles and etc. Music tracks sequence is not broken and work fine, but repeatable stuff - jingles load only once at the beginning. After the first play this files no longer to be load and play in my sequence.

"Avoid Duplicate Artists/Titles" is unchecked 
"Duplicate Prevention Time Range (Minutes)" set to 0

Below, I put a fragment of playlist exported directly from azuracast and same liquidsoap log where we can see missed jingle file#3 at the end.  

> jingle/patreon/alive.mp3
> mp3/1.0/47/01. Flite - The Wind Will Carry You.mp3
> mp3/1.0/41/17. Cartoon - Immortality (ft. Kristel A.mp3
> mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3
> jingle/silence/3.mp3
> mp3/1.0/1/04. Nu Tone - Set Me Free.mp3
> mp3/1.0/94/05. Alok & Mathieu Koss - Big Jet Plane.mp3
> mp3/1.0/23/16. Etherwood - Spoken.mp3
> mp3/1.0/95/14. Matrix & Futurebound - Human.mp3
> jingle/silence/2.mp3
> mp3/1.0/48/13. AK & Faodail - As Time Goes By.mp3
> mp3/1.0/43/20. Nitro Fun - Home.mp3
> mp3/1.0/67/04. ALEX CLARE - TOO CLOSE (THE SPEEDWAY.mp3
> mp3/1.0/42/02. Fred V & Grafix - Ultraviolet.mp3
> mp3/1.0/39/20. Galantis - Kill 'Em With The Love.mp3
> jingle/silence/3.mp3**
> mp3/1.0/73/16. Viska - Milky Way.mp3

2021/02/24 10:26:35 [playlist_03:3] Successfully loaded a playlist of 388 tracks.
2021/02/24 10:26:35 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/jingle/patreon/alive.mp3".
2021/02/24 10:26:35 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/47/01. Flite - The Wind Will Carry You.mp3".
2021/02/24 10:30:07 [lang:3] AzuraCast Raw Response: annotate:title="Immortality (ft. Kristel Aaslaid) (Futuristik Remix)",artist="Cartoon",duration="219.45",song_id="34fe5ffcf4dea04c86414ead612b21a5",media_id="3277",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/41/17. Cartoon - Immortality (ft. Kristel A.mp3
2021/02/24 10:30:07 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/41/17. Cartoon - Immortality (ft. Kristel A.mp3".
2021/02/24 10:30:46 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/41/17. Cartoon - Immortality (ft. Kristel A.mp3" (RID 12).
2021/02/24 10:30:47 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:33:46 [lang:3] AzuraCast Raw Response: annotate:title="Nectar & Ambrosia",artist="IllSkillz",duration="264.75",song_id="e37de9d0c4396f2f46c105b56029bea6",media_id="2821",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3
2021/02/24 10:33:46 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3".
2021/02/24 10:34:26 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3" (RID 13).
2021/02/24 10:34:26 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:38:11 [lang:3] AzuraCast Raw Response: annotate:title="3",duration="3.08",song_id="eccbc87e4b5ce2fe28308fd9f2a7baf3",media_id="2055",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/jingle/silence/3.mp3
2021/02/24 10:38:11 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/jingle/silence/3.mp3".
2021/02/24 10:38:14 [lang:3] AzuraCast Raw Response: annotate:title="Set Me Free",artist="Nu Tone",duration="240.12",song_id="8e63d7bbb825fd23923d28703298166b",media_id="2265",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/1/04. Nu Tone - Set Me Free.mp3
2021/02/24 10:38:14 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/1/04. Nu Tone - Set Me Free.mp3".
2021/02/24 10:38:51 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/jingle/silence/3.mp3" (RID 1).
2021/02/24 10:38:51 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:38:54 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/1/04. Nu Tone - Set Me Free.mp3" (RID 3).
2021/02/24 10:38:55 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:42:14 [lang:3] AzuraCast Raw Response: annotate:title="Big Jet Plane",artist="Alok & Mathieu Koss",duration="170.26",song_id="356bb46e9fbe2e245073344436d6e9c4",media_id="4274",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/94/05. Alok & Mathieu Koss - Big Jet Plane.mp3
2021/02/24 10:42:14 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/94/05. Alok & Mathieu Koss - Big Jet Plane.mp3".
2021/02/24 10:42:54 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/94/05. Alok & Mathieu Koss - Big Jet Plane.mp3" (RID 2).
2021/02/24 10:42:54 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:45:04 [lang:3] AzuraCast Raw Response: annotate:title="Spoken",artist="Etherwood",duration="266.68",song_id="e29fbe93b6e387c57bde45456fa3affe",media_id="2921",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/23/16. Etherwood - Spoken.mp3
2021/02/24 10:45:04 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/23/16. Etherwood - Spoken.mp3".
2021/02/24 10:45:44 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/23/16. Etherwood - Spoken.mp3" (RID 4).
2021/02/24 10:45:44 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:49:31 [lang:3] AzuraCast Raw Response: annotate:title="Human",artist="Matrix & Futurebound",duration="244.56",song_id="dc74dbb38fb2c56fe2527c893c56c156",media_id="4304",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/95/14. Matrix & Futurebound - Human.mp3
2021/02/24 10:49:31 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/95/14. Matrix & Futurebound - Human.mp3".
2021/02/24 10:50:11 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/95/14. Matrix & Futurebound - Human.mp3" (RID 1).
2021/02/24 10:50:11 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:53:36 [lang:3] AzuraCast Raw Response: annotate:title="2",duration="2.04",song_id="c81e728d9d4c2f636f067f89cc14862c",media_id="2056",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/jingle/silence/2.mp3
2021/02/24 10:53:36 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/jingle/silence/2.mp3".
2021/02/24 10:53:38 [lang:3] AzuraCast Raw Response: annotate:title="As Time Goes By",artist="AK & Faodail",duration="193.30",song_id="738745f9694240189df59e898199887e",media_id="3400",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/48/13. AK & Faodail - As Time Goes By.mp3
2021/02/24 10:53:38 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/48/13. AK & Faodail - As Time Goes By.mp3".
2021/02/24 10:54:15 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/jingle/silence/2.mp3" (RID 3).
2021/02/24 10:54:15 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:54:17 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/48/13. AK & Faodail - As Time Goes By.mp3" (RID 5).
2021/02/24 10:54:19 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 10:56:51 [lang:3] AzuraCast Raw Response: annotate:title="Home",artist="Nitro Fun",duration="204.93",song_id="446b01313fd56e5658982745908c2c63",media_id="3316",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/43/20. Nitro Fun - Home.mp3
2021/02/24 10:56:51 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/43/20. Nitro Fun - Home.mp3".
2021/02/24 10:57:31 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/43/20. Nitro Fun - Home.mp3" (RID 2).
2021/02/24 10:57:31 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 11:00:16 [lang:3] AzuraCast Raw Response: annotate:title="TOO CLOSE (THE SPEEDWAY REMIX)",artist="ALEX CLARE",duration="0.00",song_id="b8cd03e1c69ee8f48625113278421196",media_id="3762",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/67/04. ALEX CLARE - TOO CLOSE (THE SPEEDWAY.mp3
2021/02/24 11:00:16 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/67/04. ALEX CLARE - TOO CLOSE (THE SPEEDWAY.mp3".
2021/02/24 11:00:16 [lang:3] AzuraCast Raw Response: annotate:title="Ultraviolet",artist="Fred V & Grafix",duration="209.61",song_id="7a8e5a5c13984bc79c137f4351f01c1d",media_id="3283",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/42/02. Fred V & Grafix - Ultraviolet.mp3
2021/02/24 11:00:16 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/42/02. Fred V & Grafix - Ultraviolet.mp3".
2021/02/24 11:00:56 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/67/04. ALEX CLARE - TOO CLOSE (THE SPEEDWAY.mp3" (RID 4).
2021/02/24 11:00:56 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 11:05:24 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/42/02. Fred V & Grafix - Ultraviolet.mp3" (RID 6).
2021/02/24 11:05:24 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 11:08:14 [lang:3] AzuraCast Raw Response: annotate:title="Kill 'Em With The Love",artist="Galantis",duration="215.56",song_id="ba2e3de4963b845034374ac1a009b4ed",media_id="3240",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/39/20. Galantis - Kill 'Em With The Love.mp3
2021/02/24 11:08:14 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/39/20. Galantis - Kill 'Em With The Love.mp3".
2021/02/24 11:08:53 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/39/20. Galantis - Kill 'Em With The Love.mp3" (RID 1).
2021/02/24 11:08:53 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 11:11:49 [lang:3] AzuraCast Raw Response: annotate:title="Milky Way",artist="Viska",duration="147.93",song_id="c6d305aa2a554f35bd1105dcb77ac77a",media_id="3876",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/73/16. Viska - Milky Way.mp3
2021/02/24 11:11:49 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/73/16. Viska - Milky Way.mp3".
2021/02/24 11:12:29 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/73/16. Viska - Milky Way.mp3" (RID 2).
2021/02/24 11:12:29 [lang:3] AzuraCast Feedback Response: OK

**To Reproduce**

To Reproduce I fill that playlist another dummy repeatable content with a same count. Repeating is working but we can see missed files and broken sequence

> jingle/patreon/alive.mp3
> mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3
> jingle/silence/3.mp3
> jingle/silence/2.mp3
> jingle/patreon/visit.mp3
> jingle/patreon/alive.mp3
> mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3
> jingle/silence/3.mp3
> jingle/silence/2.mp3
> jingle/patreon/visit.mp3

![3](https://user-images.githubusercontent.com/44975254/108978857-1ccae700-769b-11eb-837e-600c798495a4.png)

2021/02/24 12:13:16 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/jingle/patreon/visit.mp3" (RID 8).
2021/02/24 12:13:16 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 12:13:22 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3" (RID 9).
2021/02/24 12:13:22 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 12:17:07 [lang:3] AzuraCast Raw Response: annotate:title="2",duration="2.04",song_id="c81e728d9d4c2f636f067f89cc14862c",media_id="2056",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/jingle/silence/2.mp3
2021/02/24 12:17:07 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/jingle/silence/2.mp3".
2021/02/24 12:17:09 [lang:3] AzuraCast Raw Response: annotate:title="alive",duration="11.02",song_id="7bd3d5f5b3cdb13aed632121206e729c",media_id="4456",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/jingle/patreon/alive.mp3
2021/02/24 12:17:09 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/jingle/patreon/alive.mp3".
2021/02/24 12:17:20 [lang:3] AzuraCast Raw Response: annotate:title="Nectar & Ambrosia",artist="IllSkillz",duration="264.75",song_id="e37de9d0c4396f2f46c105b56029bea6",media_id="2821",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3
2021/02/24 12:17:20 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3".
2021/02/24 12:17:47 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/jingle/silence/2.mp3" (RID 1).
2021/02/24 12:17:47 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 12:17:49 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/jingle/patreon/alive.mp3" (RID 2).
2021/02/24 12:17:49 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 12:18:00 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3" (RID 10).
2021/02/24 12:18:00 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 12:21:45 [lang:3] AzuraCast Raw Response: annotate:title="visit",duration="6.43",song_id="9de70f6546b2452f6e7b98b46ac36070",media_id="4455",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/jingle/patreon/visit.mp3
2021/02/24 12:21:45 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/jingle/patreon/visit.mp3".
2021/02/24 12:21:51 [lang:3] AzuraCast Raw Response: annotate:title="3",duration="3.08",song_id="eccbc87e4b5ce2fe28308fd9f2a7baf3",media_id="2055",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/jingle/silence/3.mp3
2021/02/24 12:21:51 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/jingle/silence/3.mp3".
2021/02/24 12:21:54 [lang:3] AzuraCast Raw Response: annotate:title="Nectar & Ambrosia",artist="IllSkillz",duration="264.75",song_id="e37de9d0c4396f2f46c105b56029bea6",media_id="2821",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3
2021/02/24 12:21:54 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3".
2021/02/24 12:22:25 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/jingle/patreon/visit.mp3" (RID 3).
2021/02/24 12:22:25 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 12:22:31 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/jingle/silence/3.mp3" (RID 4).
2021/02/24 12:22:31 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 12:22:34 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3" (RID 5).
2021/02/24 12:22:34 [lang:3] AzuraCast Feedback Response: OK
2021/02/24 12:26:19 [lang:3] AzuraCast Raw Response: annotate:title="Nectar & Ambrosia",artist="IllSkillz",duration="264.75",song_id="e37de9d0c4396f2f46c105b56029bea6",media_id="2821",liq_amplify="0.00dB",playlist_id="58":/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3
2021/02/24 12:26:19 [decoder:3] Method "FFMPEG" accepted "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3".
2021/02/24 12:26:59 [dnb_edm_next_song:3] Prepared "/var/azuracast/stations/dnb_edm/media/mp3/1.0/18/05. IllSkillz - Nectar & Ambrosia.mp3" (RID 6).
2021/02/24 12:26:59 [lang:3] AzuraCast Feedback Response: OK

I honestly don’t know how else to catch this annoying bug... Please help, I can provide any other necessary information

**Comment by Abqehebter** [comment]
Yup. I have the exact same problem, only it's not as important with me probably. In my case, the advertising triggers are supposed to run via order, but in reality it usually runs like this:

Adtrigger1, Adtrigger2, Adtrigger4, Adtrigger5, Adtrigger2 ....

**Comment by SlvrEagle23** [comment]
Closing in favor of #3863 which likely represents the exact same issue.
